From: Simon Quigley <tsimonq2@ubuntu.com>
Date: Tue, 7 Apr 2026 20:09:09 +0800
Subject: Show list of users by default

Origin: vendor
Last-Update: 2025-01-07

Just because a config entry says that it's false by default does not mean it
actually is.
Last-Update: 2025-01-07
---
 data/lightdm.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/lightdm.conf b/data/lightdm.conf
index 0df3842..0d5be9a 100644
--- a/data/lightdm.conf
+++ b/data/lightdm.conf
@@ -100,9 +100,9 @@
 #xdmcp-port=177
 #xdmcp-key=
 #greeter-session=example-gtk-gnome
-#greeter-hide-users=false
+greeter-hide-users=false
 #greeter-allow-guest=true
-#greeter-show-manual-login=false
+greeter-show-manual-login=false
 #greeter-show-remote-login=true
 #user-session=default
 #allow-user-switching=true
